### ISSUE TITLE
The connection to the camera is force closed automatically after around 2 minutes

### ISSUE THREAD

**Issue by dinhchinh82** [linkwil/WeHomeSDK#8]
Hello,

We are trying to integrate your camera to our own application. As the test result, the connection to the camera is force closed automatically after around 2 minutes viewing from the demo app.

We also tested with your official app and got the same issue.

What we expected is how to keep the connection as soon as end-user still open the viewing screen.

Please suggest us the solution how to do.

Thank you.

**Comment by xiaojinsheng** [comment]
Yes, the limitation of connection time for each session is 2 minutes, it is designed for avoid unexpected power consumption.
We suggest you add a countdown timer on the viewing screen, and popup some alert to customer when countdown timer is 0, actually, we are also planning to do the same thing on WeHome APP

**Comment by dinhchinh82** [comment]
the limitation of 2 minutes is applied to all the devices or not? I mean, for the camera which is attached to the solar panel, the power consumption may not be critical so it is OK to keep the connection longer. How do you think?

**Comment by xiaojinsheng** [comment]
Currently, it is for all the devices.
I will report your opinion to my college to check if we should modify something. but for now, you can keep 2 minutes limitation for all devices, we will notify you if we changed something.
